packages/slas-react-sdk/src/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { createContext, useContext } from "react";
+import { createStore, useStore } from "zustand";
```

Comment:
What is reason behind using this library over other state management ones?

Reply:
please read the **Implementation detail** section in the PR description.